fast_llm/models/gpt/model.py

```diff
@@ -71,7 +71,35 @@ def __init__(
         else:
             self._flash_varlen_preprocessor = FlashAttnVarlenPreprocessor(self._config.transformer, self._tensor_space)
 
+    def get_output_layers(self) -> list[Layer]:
+        return [
+            layer
+            for i in range(self._config.prediction_heads)
+            for layer in [
+                TransformerLayer(
+                    self._config.transformer,
+                    self._tensor_space,
+                    # TODO MTP: which index?
+                    layer_index=self._config.transformer.num_layers,
+                    # The last layer only returns the transformer output.
+                    # The previous layers return a stack of shared_hidden and transformer_output.
+                    return_input=i < self._config.prediction_heads - 1,
+                ),
+                LanguageModelHead(
+                    self._config,
+                    self._tensor_space,
+                    prediction_distance=i,
+                ),
+            ]
+        ]
+
     def get_layers(self) -> list[Layer]:
+        if self._config.transformer.num_layers == 0:
+            Assert.eq(self._config.prediction_heads, 1)
```

Comment:
This could be checked in config validation

Reply:
for a debug case like num_layers=0 that has no practical applications?
I think not.